app/routes/image-set.js

```diff
@@ -15,5 +15,45 @@ export default Ember.Route.extend( OrganizationRouteMixin, RequireUserMixin, {
 
   model: function(params) {
     return this.store.find('imageSet', params.image_set_id);
+  },
+
+  actions: {
+    associate: function(imageSet, lion) {
+      var route = this,
+          previousIsVerified = imageSet.get('isVerified');
+
+      imageSet.setProperties({
+        lion: lion,
+        isVerified: false
+      });
+
+      imageSet.save().then(function() {
+        route.transitionTo('lion', lion);
+      }, function(){
+        imageSet.setProperties({
```

Comment:
probably don't want to just blindly throw away errors here — they might be legitimate API or network failures?